.circleci/config.yml

```diff
@@ -371,8 +819,8 @@ workflows:
                 - master
                 - production
 
-  # *************** back ***************
-  back:
+  # BACK-END ESSENTIAL
+  back-essential:
     when: << pipeline.parameters.back >>
```

Comment:
I think this also needs the same conditions as the back-ee workflow.  Could be I'm missing something, but I suspect it will run again when deploying, otherwise.
```
    when:
      and:
        - not: <<pipeline.parameters.run_deploy>>
        - << pipeline.parameters.back >>
```

Reply:
Good catch 🎣